client/components/themes-list/index.jsx

```diff
@@ -40,6 +40,26 @@ const getGridColumns = ( gridContainerRef, minColumnWidth, margin ) => {
 	return columnsPerRow;
 };
 
+const getSiteAssemblerUrl = ( {
+	isLoggedIn,
+	selectedSite,
+	shouldGoToAssemblerStep,
+	siteEditorUrl,
+} ) => {
+	if ( ! shouldGoToAssemblerStep ) {
```

Comment:
We should keep the `isLoggedIn` check here since `siteEditorUrl` will be invalid for logged-out mobile view:

![Screenshot 2023-07-27 at 11 32 36 AM](https://github.com/Automattic/wp-calypso/assets/797888/379cffe4-9d67-4afc-ab2f-66f781bae6a2)

Clicking this CTA in the LoTS (mobile) will redirect users to: `https://container-interesting-dubinsky.calypso.live/nullsite-editor.php?canvas=edit&assembler=1&calypso_origin=https%3A%2F%2Fcontainer-interesting-dubinsky.calypso.live`

On production, it redirects users to: `https://wordpress.com/start/with-theme-assembler`

Reply:
I swear I had this checked but got lost somewhere during the refactor 😂 Fixed, thanks for checking!